figures/README.md

```diff
@@ -178,3 +178,55 @@ The script can be called from anywhere in this repository (will look for the `.g
 The hex codes table in `figures/README.md` and its swatches should also be updated by using the `swatches_table` function at the end of the script and copy and pasting this function's output to the appropriate place in the table.
 
 The histology color palette file is created by running `Rscript -e "rmarkdown::render('figures/mapping-histology-labels.Rmd', clean = TRUE)"`.
+
+
+### Overall figure theme
+
+In general, we will use the `ggpubr` package with `ggtheme = theme_pubr())` and color palette `simpsons` from package `ggsci` since it has 16 levels and can accommodate the levels in groups such as `molecular_subtype`.
+
+To view the palette:
+```
+library("scales")
+show_col(pal_simpsons("springfield")(16))
```

Comment:
This will not just work ™️  in the project container as is, so:

```suggestion
scales::show_col(ggsci::pal_simpsons("springfield")(16))
```

Reply:
thanks!